modules/virtualization/microvm/idsvm.nix

```diff
@@ -0,0 +1,167 @@
+# Copyright 2022-2023 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}: let
+  configHost = config;
+  vmName = "ids-vm";
+  macAddress = "02:00:00:01:01:02";
+  networkName = "ethint0";
+  idsvmBaseConfiguration = {
+    imports = [
+      # (import ./common/vm-networking.nix {inherit vmName macAddress useDHCP;})
```

Comment:
This can be cleaned as it's not in use.

Reply:
Resolved by [0e2656b](https://github.com/tiiuae/ghaf/commit/0e2656bf1a00ef6752bcbf2781aa5ae7f204b685)